build-tests/api-extractor-scenarios/src/excerptTokens/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// prettier-ignore
```

Comment:
I was able to eliminate the `api-extractor-test-05`  project by using `// prettier-ignore` in the .d.ts file and adding a step to `runScenarios.ts` that copies the .d.ts files to `lib/`

@zelliott